### PR TITLE
Remove menubar and reposition edit button

### PR DESCRIPTION
**Pull request by fernandojsg**
Re https://github.com/aframevr/aframe-editor/issues/217:
screenshot 2016-07-20 22 56 26

### CODE REVIEW

File: src/components/Main.js [hunk 1]
```diff
@@ -41,16 +40,9 @@ export default class Main extends React.Component {
...
 
-  toggleEditor = () => {
-    this.setState({editorEnabled: !this.state.editorEnabled}, function () {
-      if (this.state.editorEnabled) {
-        editor.enable();
-      } else {
-        editor.disable();
-      }
-      // Events.emit('editorModeChanged', this.state.editorEnabled);
+    Events.on('editorModeChanged', (enabled) => {
```
Comment (ngokevin) [inline src/components/Main.js]: don't need parens

File: src/components/components/Sidebar.js [hunk 2]
```diff
@@ -16,7 +16,7 @@ export default class Sidebar extends React.Component {
   render () {
     return (
       <div id='sidebar'>
-        <div className='sidebar-title'>components</div>
+        <div className='sidebar-title'>Attributes</div>
```
Comment (ngokevin) [inline src/components/components/Sidebar.js]: Keep as Components

File: src/css/main.css [hunk 3]
```diff
@@ -864,18 +878,17 @@ code, pre {
 }
 
 .sidebar-title {
-  color: #1faaf2;
-  text-align: center;
-  background-color: #323232;
-  padding: 2px;
-  border-top: 2px solid #1faaf2;
-  font-size: 10px;
+	color: #AAA;
```
Comment (ngokevin) [inline src/css/main.css:881]: let's do 2-space indents and no tab

**Comment by ngokevin** [comment]
r+wc

**Comment by fernandojsg** [comment]
Done! thank you!

**Pull request merged**